build/build.go

```diff
@@ -91,6 +92,29 @@ func Watch(extension core.Extension, report ResultHandler) {
 	logProcessors.Wait()
 }
 
+func evaluateTemplate(rawTemplate string, ext core.Extension, ctx core.IntegrationContext) string {
+	type contextRoot struct {
+		core.Extension
+		core.IntegrationContext
+		BundleUrls []string
+	}
+
+	var buf bytes.Buffer
+	bundleUrls := []string{}
+	for _, asset := range ext.Development.Entries {
+		bundleUrls = append(bundleUrls, ctx.PublicUrl + "/" + asset)
```

Comment:
Functionally, this loop works, but I'm not sure the source fields are correct:

    `https://example.ngrok.io/src/index.tsx`